cli/tests/test_checkout.rs

```diff
@@ -100,6 +100,38 @@ fn test_checkout_not_single_rev() {
     "###);
 }
 
+#[test]
+fn test_checkout_conflicting_change_ids() {
+    let (test_env, repo_path) = set_up();
```

Comment:
As with the parent commit, I don't think we need to involve a remote here. You can do it like this:

```
jj init --git
jj describe -m one
jj --at-op=@- describe -m two
```